test/parse_test.rb

```diff
@@ -3,7 +3,11 @@
 require "yarp_test_helper"
 
 # It is useful to have a diff even if the strings to compare are big
-Test::Unit::Assertions::AssertionMessage.max_diff_target_string_size = 5000
+# However, ruby/ruby does not have a version of Test::Unit with access to
+# max_diff_target_string_size
+if defined?(Test::Unit::Assertions::AssertionMessage)
```

Comment:
Seems like CRuby doesn't have access to `AssertionMessage`. Not sure why `tool/lib/test/unit/assertions.rb` seems dated, but this snippet is a debugging optimization anyway, so not necessary in the sync